qoqo_myqlm/interface/utilities.py

```diff
@@ -0,0 +1,27 @@
+# Copyright © 2019-2024 HQS Quantum Simulations GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+import numpy as np
+
+
+def generate_VariableMSXX_matrix(theta):
```

Comment:
I would suggest adding type annotations to this function, for the user to know if one can pass a CalculatorFloat or just a float.